wally-registry-frontend/src/pages/Package.tsx

```diff
@@ -173,150 +179,188 @@ export default function Package() {
                   <a href={packageMetadata?.package.registry}>
                     {packageMetadata?.package.registry.replace("https://", "")}
                   </a>
                 </MetaItem>
               )} */}
 
-                {packageMetadata?.package.authors &&
-                  packageMetadata?.package.authors.length > 0 && (
-                    <MetaItem title="Authors" width="full">
-                      {packageMetadata?.package.authors.map((author) => (
-                        <p key={author}>{author}</p>
-                      ))}
-                    </MetaItem>
-                  )}
-
-                {packageMetadata?.dependencies &&
-                  Object.values(packageMetadata?.dependencies).length > 0 && (
-                    <MetaItem title="Dependencies" width="full">
-                      {Object.values(packageMetadata?.dependencies).map(
-                        (dependency) => (
-                          <DependencyLink
-                            key={dependency}
-                            packageInfo={dependency}
-                          />
-                        )
-                      )}
-                    </MetaItem>
+            {packageMetadata?.package.authors &&
+              packageMetadata?.package.authors.length > 0 && (
+                <MetaItem title="Authors" width="full">
+                  {packageMetadata?.package.authors.map((author) => (
+                    <p key={author}>{author}</p>
+                  ))}
+                </MetaItem>
+              )}
+
+            {packageMetadata?.dependencies &&
+              Object.values(packageMetadata?.dependencies).length > 0 && (
+                <MetaItem title="Dependencies" width="full">
+                  {Object.values(packageMetadata?.dependencies).map(
+                    (dependency) => (
+                      <DependencyLink
+                        key={dependency}
+                        packageInfo={dependency}
+                      />
+                    )
                   )}
-
-                {packageMetadata &&
-                  packageMetadata["server-dependencies"] &&
-                  Object.values(packageMetadata["server-dependencies"]).length >
-                    0 && (
-                    <MetaItem title="Server Dependencies" width="full">
-                      {Object.values(
-                        packageMetadata["server-dependencies"]
-                      ).map((dependency) => (
-                        <DependencyLink
-                          key={dependency}
-                          packageInfo={dependency}
-                        />
-                      ))}
-                    </MetaItem>
+                </MetaItem>
+              )}
+
+            {packageMetadata &&
+              packageMetadata["server-dependencies"] &&
+              Object.values(packageMetadata["server-dependencies"]).length >
+                0 && (
```

Comment:
Can be simplified since `server-dependencies` isn't null, nor is packageMetadata. Also, in particular, there is no need to use `Object.values` to check a length of an array.
```suggestion
            {packageMetadata["server-dependencies"].length > 0 && (
```